package.json

```diff
@@ -29,6 +29,7 @@
     "/directives",
     "/generated",
     "/helpers",
+    "/lang",
```

Comment:
There are a few other places we list out all the directories (like linting). Really wish we'd made a top-level `src` but too late for that!